service-base/src/test/java/org/eclipse/hono/service/credentials/CompleteBaseCredentialsServiceTest.java

```diff
@@ -302,6 +303,11 @@ public void getAll(final String tenantId, final String deviceId, final Handler<A
             protected int getMaxBcryptIterations() {
                 return MAX_ITERATIONS;
             }
+
+            @Override
+            public Vertx getVertx() {
+                return Vertx.vertx();
```

Comment:
this creates a new vert.x instance on each invocation. are you sure this is what you want?

Reply:
Right. I pushed a fix.